sway/tree/view.c

```diff
@@ -709,7 +709,7 @@ void view_update_size(struct sway_view *view, int width, int height) {
 		con->surface_y = fmax(con->surface_y, con->content_y);
 	}
 	con->surface_width = width;
-	con->surface_width = height;
+	con->surface_height = height;
```

Comment:
Please double-check this. It looks like a copy-mistake to me, but it's possible that I don’t know what I'm doing.

Reply:
I agree that this is a copy/paste mistake.